pkg/controller/podgc/gc_controller_test.go

```diff
@@ -295,6 +295,7 @@ func TestGCOrphaned(t *testing.T) {
 			},
 			itemsInQueue:    1,
 			deletedPodNames: sets.NewString("a"),
+			patchedPodNames: sets.NewString("a"),
```

Comment:
Why do you think this was needed?  Since feature was turned on, we are patching more because of the terminating status?

Reply:
I think we patch more becasue we have this check in PodGC Controller - https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/podgc/gc_controller.go#L349

Reply:
Yea I remember that change now.